src/main/java/me/coley/j2h/loader/Importer.java

```diff
@@ -0,0 +1,48 @@
+package me.coley.j2h.loader;
+
+import me.coley.j2h.modle.*;
+
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBException;
+import javax.xml.bind.Unmarshaller;
+import java.io.IOException;
+import java.io.StringReader;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public final class Importer {
+
+    public static Configuration importConfiguration(String path) throws JAXBException, IOException {
+        JAXBContext context = JAXBContext.newInstance(Configuration.class);
+        Unmarshaller um = context.createUnmarshaller();
+        return (Configuration) um.unmarshal(new StringReader(readFile(path, StandardCharsets.UTF_8)));
+    }
+
+    public static Configuration importDefaultConfiguration() throws JAXBException, IOException {
+        return importConfiguration(Importer.class.getClassLoader().getResource("default-config.j2h").getPath());
+    }
+
+    private static String readFile(String path, Charset encoding) throws IOException {
+        byte[] encoded = Files.readAllBytes(Paths.get(path));
```

Comment:
Running in IntelliJ results in the following:
```
Caused by: java.nio.file.InvalidPathException: Illegal char <:> at index 2: /D:/Programming/Java/java2html/target/classes/default-config.j2h
	at sun.nio.fs.WindowsPathParser.normalize(WindowsPathParser.java:182)
	...
	at java.nio.file.Paths.get(Paths.java:84)
	at me.coley.j2h.loader.Importer.readFile(Importer.java:28)
	at me.coley.j2h.loader.Importer.importConfiguration(Importer.java:20)
	at me.coley.j2h.loader.Importer.importDefaultConfiguration(Importer.java:24)
```
Using an alternative to `Files.readAllBytes` to read a class-path resource should resolve this.

Reply:
Running it on Ubuntu with Intellij works for me. As I cannot reproduce this issue. Can you find a fix?

Reply:
Leading `/` seems to be the issue. 
```java
public static Configuration importDefaultConfiguration() throws JAXBException, IOException, URISyntaxException {
    ClassLoader classloader = Importer.class.getClassLoader();
    URI uri = classloader.getResource("default-config.j2h").toURI();
    Path path = Paths.get(uri);
    return importConfiguration(path.toString());
}
```
One possible fix is `Paths.get(URI)` instead of `Paths.get(String)`. 

And if you want to get rid of that new `URISyntaxException` the following works:
```java
public static Configuration importDefaultConfiguration() throws JAXBException, IOException {
	ClassLoader classloader = Importer.class.getClassLoader();
	String uri = classloader.getResource("default-config.j2h").toExternalForm();
	Path path = Paths.get(URI.create(uri));
	return importConfiguration(path.toString());
}
```

Reply:
Done. I have tested this now one Windows and Ubuntu.